core/templates/dev/head/pages/exploration-player-page/services/exploration-player-state.service.ts

```diff
@@ -68,15 +68,28 @@ oppia.factory('ExplorationPlayerStateService', [
     var editorPreviewMode = ContextService.isInExplorationEditorPage();
     var questionPlayerMode = ContextService.isInQuestionPlayerMode();
     var explorationId = ContextService.getExplorationId();
-    var version = GLOBALS.explorationVersion;
+    var version = UrlService.getExplorationVersionFromUrl();
+    if (version) {
+      ReadOnlyExplorationBackendApiService
+        .loadExploration(explorationId, version)
+        .then(function(exploration) {
+          version = exploration.version;
+        });
+    } else {
+      ReadOnlyExplorationBackendApiService
+        .loadExploration(explorationId, version)
+        .then(function (exploration) {
+          version = exploration.version;
+        });
+    }
```

Comment:
Why are these two the same?

Reply:
Changed.
The second was supposed to load the exploration without the version (latest exploration).